backend/src/data/repositories/tag.repository.ts

```diff
@@ -16,6 +20,7 @@ class TagRepository extends Repository<Tag> {
       .orderBy('tag.name', 'ASC')
       .getMany();
   }
```

Comment:
Just remove `.addSelect('tag.workspaceId')` and as return you will get `Promise:<ITag[]>` no need for further mapping

```suggestion
public findAllByWorkspaceId(workspaceId: string): Promise<ITag[]> {
    return this.createQueryBuilder('tag')
      .select('tag.id')
      .addSelect('tag.name')
      .where('tag.workspaceId= :workspaceId', { workspaceId })
      .orderBy('tag.name', 'ASC')
      .getMany();
  }
  }
```